DataModels/VM/Scheduler/SchedulerVM.cs

```diff
@@ -120,6 +122,8 @@ public SchedulerVM()
         public List<DropDownLargeValues> Member1List { get; set; }
         public List<DropDownLargeValues> Member2List { get; set; }
 
+        public List<DropDownLargeValues> FlightTagsList { get; set; }
```

Comment:
Not required large values. use DropDownValues. ignore documenttag for this

Reply:
Not Resolved